CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 ### Changed
+- Fixed issue: #236
```

Comment:
Please either add a hyperlink to the issue or 1 sentence describing what was fixed